source/index.ts

```diff
@@ -260,7 +260,7 @@ is.primitive = (value: unknown): value is Primitive => is.null_(value) || isPrim
 is.integer = (value: unknown): value is number => Number.isInteger(value as number);
 is.safeInteger = (value: unknown): value is number => Number.isSafeInteger(value as number);
 
-is.plainObject = <Value = unknown>(value: unknown): value is Record<string, Value> => {
+is.plainObject = <Value = unknown>(value: unknown): value is Record<string | number | symbol, Value> => {
```

Comment:
```suggestion
is.plainObject = <Value = unknown, Key = string | number | symbol>(value: unknown): value is Record<Key, Value> => {
```

Just for cases people know that they might have just strings, and it wouldn't mess with their types. Default behavior would change (as this is a breaking change, sort of), but people would be able to easily decide the key type.

Reply:
Interesting, is there any difference between `keyof any` and `string | number | symbol`?

Reply:
> Interesting, is there any difference between keyof any and string | number | symbol?

@gioragutt ^

Reply:
Should we maybe move `string | number | symbol` to a `type` called `ObjectKey` since it's used in multiple places?

Reply:
AFAIR `keyof any === string |number | symbol`

Reply:
Then I would prefer `string | number | symbol`.

Reply:
So we should stick with the original commit where it was `string | number | symbol` ?

Reply:
> Should we maybe move string | number | symbol to a type called ObjectKey since it's used in multiple places?

I need opinions on this.

Reply:
Yeah, that would be pretty handy in my opinion.

Reply:
Let's do that then.

Reply:
@sindresorhus  The suggested `ObjectKey` type is added.